estate/__manifest__.py

```diff
@@ -0,0 +1,12 @@
+{
+    'name': 'RealEstate',
+    'version': '0.1',
+    'summary': 'Tracks RealEstate property and buyer offers',
+    'description': "Tracks RealEstate property and buyer offers",
+    'depends': [
+        'base'
+    ],
+    'installable': True,
+    'application': True,
+    'auto_install': False
```

Comment:
`installable` and `auto_install` are by default these values, no need to repeat it.
```suggestion
    'depends': ['base'],
    'application': True,
```